.github/workflows/build.yml

```diff
@@ -5,6 +5,11 @@ on:
   push:
     branches:
     - main
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
   pull_request:
     branches:
     - main
```

Comment:
I haven't worked a lot with filters and trigger types etc but this doesn't look how I'd expect it to.
To me this reads like:
We run the workflow if anything is pushed to main, and I'm not sure how to interpret these triggers in this case.
We also run the workflow on PRs when their base is main (using the default triggers as nothing is specified under this).

In other words, shouldn't the `types` element and it's children be under `pull_request` instead of under `push`?

Reply:
Yep, good point, I will fix this later today 👍